safe/processors/__init__.py

```diff
@@ -40,6 +40,9 @@
     post_processor_fatalities,
     post_processor_displaced_ratio,
     post_processor_displaced,
+    post_processor_distance,
+    post_processor_bearing,
+    post_processor_cardinality
```

Comment:
Can you update the kind of tree a few lines before? Cardinality is a child of bearing because it's using the output of the bearing output.

Reply:
Thanks, for reminding me